src/tool_help.c

```diff
@@ -279,6 +279,8 @@ static const struct helptxt helptext[] = {
    "Write to file instead of stdout"},
   {"-Z, --parallel",
    "Perform transfers in parallel"},
+  {"    --parallel-connect",
+   "Prefer parallel connections to multiplexed ones"},
```

Comment:
Regardless of what this option is named, I think the description should add `use with --parallel` because it's somewhat ambiguous like we had with --retry and --retry-connrefused, for example

~~~
"Prefer new connections over multiplex (use with --parallel)"
~~~

Reply:
Or optionally, let this option imply `--parallel` at the same time..

Reply:
Saw the tweet so I’m here to try making a suggestion. Not really familiar with naming convention for options but if concern is readability, why not `—parallel-with-new-connect` ?

Reply:
Well, it isn't unconditionally a new connect. It just will *prioritize* speed (ie at times a new connection) before connection re-use. It will still multiplex if there already is an existing connection available to use for that...